src/components/blog-item.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+import blogStyles from '../styles/blog.module.css'
+
+interface blogProps {
```

Comment:
```suggestion
interface BlogItemProps {
```

More consistent this way, so it corresponds to the component name. What do you think? Also thanks for using typescript!